### PR TITLE
Upgrade musgo to v0.1.11.

### PR DESCRIPTION
**Pull request by ymz-ncnk**

### CODE REVIEW

**Review by alecthomas — APPROVED** [review]

**Comment by alecthomas** [comment]
Can you update the README with the new results please?

**Comment by alecthomas** [comment]
(unless you don't care about adding them in which case, that's fine too)

File: MusgoAUnsafe.musgen.go [hunk 1]
```diff
@@ -8,12 +8,18 @@ import (
 	"github.com/ymz-ncnk/musgo/errs"
 )
 
-func (v NoTimeA) MarshalMUSUnsafe(buf []byte) int {
+// MarshalMUSUnsafe fills buf with the MUS encoding of v.
```
[the remark this answers is not in the document]
Comment (ymz-ncnk) [reply MusgoAUnsafe.musgen.go:13]: Done

**Comment by ymz-ncnk** [comment]
README already has new results

**Comment by alecthomas** [comment]
Did you close it accidentally?

**Comment by ymz-ncnk** [comment]
No, I'll make new one, soon.
________________________________

**Pull request closed without merging**